pkg/sql/lease_internal_test.go

```diff
@@ -581,9 +581,11 @@ CREATE TABLE t.test (k CHAR PRIMARY KEY, v CHAR);
 //    acquireFreshestFromStore() or acquire() notices, after re-acquiring the
 //    tableState lock, that the new lease has been released and acquires a new
 //    one.
-func TestLeaseAcquireAndReleaseConcurrenctly(t *testing.T) {
+func TestLeaseAcquireAndReleaseConcurrently(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 
+	t.Skip("fails in the presence of migrations requiring backfill, but cannot import sqlmigrations")
```

Comment:
@lego: I had to disable this one as I can't even use the `sqlmigrations` testing knobs to disable it due to dependency cycles. I tried moving it to the `sql_test` package, but of course there are too many private methods used to do that.

Reply:
👍 that's fine for now. I'll see if I can move or re-evaluate the test after this PR. Sorry for the trouble!